docs/content/usages/promise.md

```diff
@@ -0,0 +1,87 @@
+---
+sort: 2
+---
+
+# Action Promise
+
+Nikita's actions always return [Javascript Promise](https://nodejs.dev/learn/understanding-javascript-promises). To access the action output or to run JavaScript commands synchronously, you have to call an asynchronous function and ["await"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await) for the result from Promise:
```

Comment:
you don't have to call an async function, even you example is based on a sync function

Reply:
changed